R/select_rows.r

```diff
@@ -17,12 +20,14 @@
 #' select_rows(iris, c(TRUE,FALSE)) # Select only odd rows
 #' iris2 <- iris; rownames(iris2) <- paste0("row", 1:nrow(iris2))
 #' select_rows(iris, c("row10", "row51")) # Select rows by name
-select_rows <- function(dataframe, rows, whole = FALSE, ...) {
+select_rows <- function(dataframe, rows, whole = FALSE, cols_to_check = NULL, ...) {
```

Comment:
Can you move the argument to the right of the `...`? We want to preserve backward compatibility with function calls.